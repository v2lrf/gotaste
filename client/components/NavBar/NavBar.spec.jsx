import React from 'react'
import { NavBar } from '.'

describe('NavBar', () => {
  let component
  beforeEach(() => {
    component = shallow(<NavBar />)
  })

  describe('Snapshots', () => {
    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
