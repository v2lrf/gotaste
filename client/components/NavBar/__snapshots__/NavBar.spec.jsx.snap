// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Snapshots should match the snapshot 1`] = `
<div
  className="bg-white border-b border-grey-light"
>
  <div
    className="bg-red-light h-1"
  />
  <Container
    narrow={false}
  >
    <div
      className="h-auto sm:h-16 flex flex-col sm:flex-row items-center sm:justify-between"
    >
      <div
        className="my-2 sm:my-0"
      >
        <Link
          href="/"
        >
          <a
            className="text-red-dark text-2xl font-bold no-underline"
          >
            govinu
          </a>
        </Link>
      </div>
      <nav
        className="pb-4 pt-1 sm:pb-0 sm:pt-0 overflow-hidden max-w-full"
      >
        <ul
          className="list-reset flex overflow-x-auto flex-row whitespace-no-wrap text-sm sm:text-base"
        >
          <NavItem
            to="/discover"
          >
            Gå på opdagelse
          </NavItem>
          <NavItem
            to="#"
          >
            Begivenheder
          </NavItem>
          <NavItem
            to="#"
          >
            Forhandlere
          </NavItem>
          <Query
            query={
              Object {
                "definitions": Array [
                  Object {
                    "directives": Array [],
                    "kind": "OperationDefinition",
                    "name": Object {
                      "kind": "Name",
                      "value": "Viewer",
                    },
                    "operation": "query",
                    "selectionSet": Object {
                      "kind": "SelectionSet",
                      "selections": Array [
                        Object {
                          "alias": undefined,
                          "arguments": Array [],
                          "directives": Array [],
                          "kind": "Field",
                          "name": Object {
                            "kind": "Name",
                            "value": "viewer",
                          },
                          "selectionSet": Object {
                            "kind": "SelectionSet",
                            "selections": Array [
                              Object {
                                "alias": undefined,
                                "arguments": Array [],
                                "directives": Array [],
                                "kind": "Field",
                                "name": Object {
                                  "kind": "Name",
                                  "value": "shortName",
                                },
                                "selectionSet": undefined,
                              },
                            ],
                          },
                        },
                      ],
                    },
                    "variableDefinitions": Array [],
                  },
                ],
                "kind": "Document",
                "loc": Object {
                  "end": 57,
                  "start": 0,
                },
              }
            }
          >
            <Component />
          </Query>
        </ul>
      </nav>
    </div>
  </Container>
</div>
`;
